pkg/splunk/client/enterprise.go

```diff
@@ -470,7 +470,7 @@ func (c *SplunkClient) GetIndexerClusterPeerInfo() (*IndexerClusterPeerInfo, err
 	return &apiResponse.Entry[0].Content, nil
 }
 
-// ClusterMasterPeerInfo represents the status of a indexer cluster peer (cluster master endpoint).
+// ClusterMasterPeerInfo represents the status of a indexer cluster peer (cluster manager endpoint).
```

Comment:
Do we want to take care of these struct/function names too?

Reply:
Great point Kriti. These cases will be taken care of in future PRs. The reason is that the Functions/Structs themselves will have their name changed. A few more PRs coming in the future :D. For now, we'll keep these out of scope.